model/service.go

```diff
@@ -54,6 +54,11 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// For the service that runs on VMs, ServiceAccountsOnVm specifies the service
+	// accounts provisioned to these VMs. When multiple service accounts are
+	// specified, use comma as the delimiter.
+	ServiceAccountsOnVm string `json:"serviceaccountsonvm,omitempty"`
```

Comment:
Drive by comment. This is very clunky and visually breaks the platform agnostic service model. Do we need to differentiate the service account on vm from service account on k8s ?